### DISCUSSION TITLE
Allow copying of blocks between transforms

### DISCUSSION THREAD

**Discussion by mkiang** [redivis/meta#178] [Feature requests]
**What is the problem you'd like us to address**
It would be nice to be able to copy a single block (e.g., a join or new variable) across transforms. 

**Describe a solution (if you have one in mind)**
Unknown. I tried to copy the relevant part of the code that corresponded with my block of interest into a new transform but it didn't quite work how I had hoped (i.e., the interface wasn't appropriately updated). 

**Describe other related products we could check out (if applicable)**

**Any additional context**

**Comment by sfmcintyre** [comment]
Implemented in our [latest release](https://redivis.com/updates#2023-06-22)
